addons/payment_adyen/views/payment_acquirer.xml

```diff
@@ -7,11 +7,12 @@
             <field name="model">payment.acquirer</field>
             <field name="inherit_id" ref="payment.acquirer_form"/>
             <field name="arch" type="xml">
-                <xpath expr='//group[@name="acquirer_display"]' position='after'>
-                    <group attrs="{'invisible': [('provider', '!=', 'adyen')]}">
+                <xpath expr='//group[@name="payment_acquirer"]' position='inside'>
+                    <group attrs="{'invisible': [('provider', '!=', 'adyen')]}" class="mt0">
```

Comment:
As current scenario creates group within group.Group will create table and it will add margin to table so group within group will create extra margin.So for inside group i will use mt0(margin-top:0px)